proposals/specification/access-consents.bs

```diff
@@ -766,6 +845,59 @@ Jose's [=Agent Registry=].
   </pre>
 </figure>
 
+When a [=Data Grant=] is assigned the `interop:SelectedFromRegistry` scope, the
+`interop:registeredAgent` is authorized to access only the specific
+[=Data Instances=] linked via `interop:hasDataInstance` in the [=Data Registration=]
+linked via `interop:hasDataRegistration`,
+limited by the [=access modes=] linked via `interop:accessMode`.
+
+<figure>
+  <figcaption>Valid [=access modes=] for a [=Data Instance=]</figcaption>
+  <table class="data tree" align="left">
+    <col>
+    <col>
+    <thead>
+      <tr>
+        <th>Mode</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>`acl:Create`</td>
+        <td>Create new resources within the [=Data Instances=] where applicable. 
+        [=Access modes=] assigned to 
+        the creator on a created resource should be a union of 
+        `interop:accessMode` and `interop:createAccessMode` modes. 
+        <b>acl:Create is not currently supported. 
+        Substitute with acl:Write or acl:Append, but may exceed intended scope
+        of access.</b></td>
+      </tr>
```

Comment:
```suggestion
```

Currently, we don't support *Create* with *SelectedFromRegistry* scope. We probably should create issue to track or just add note that *Create* is not supported.

Reply:
@elf-pavlik In this context it would giving the grantee the ability to create resources within the Data Instance, not create new data instances on the data registration.

Reply:
ok